gamemodes/terrortown/entities/entities/ttt_health_station.lua

```diff
@@ -181,34 +173,12 @@ if SERVER then
 
 	---
 	-- @realm server
-	local ttt_damage_own_healthstation = CreateConVar("ttt_damage_own_healthstation", "0", FCVAR_NONE, "0 as detective cannot damage their own health station")
```

Comment:
mention this in the changelog

Reply:
This convar was already inaccessible as it wasn't in the UI or documentation. But I can add it, sure